File: eCopy.Model/Response/PersonResponse.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace eCopy.Model.Response
{
    public class PersonResponse
    {
        public int ID { get; set; }
        public string FirstName { get; set; }
        public string LastName { get; set; }
        public string MiddleName { get; set; }
        public string Gender { get; set; }
        public int CityId { get; set; }
        public CityResponse City { get; set; }
        public string Address { get; set; }
        public DateTime BirthDate { get; set; }
    }
}

File: ecopymobile/myapp/lib/model/registrationModel/response/personResponse.dart
import 'dart:ffi';

import 'package:json_annotation/json_annotation.dart';
import 'package:myapp/model/registrationModel/response/cityResponse.dart';

part 'personResponse.g.dart';

@JsonSerializable()
class PersonResponse {
  int? id;
  String? firstName;
  String? lastName;
  String? middleName;
  String? gender;
  int? cityId;
  CityResponse? city;
  String? address;
  DateTime? birthDate;

  PersonResponse() {}

  factory PersonResponse.fromJson(Map<String, dynamic> json) =>
      _$PersonResponseFromJson(json);

  /// Connect the generated [_$PersonResponseToJson] function to the `toJson` method.
  Map<String, dynamic> toJson() => _$PersonResponseToJson(this);
}
